spec/factories/users.rb

```diff
@@ -51,6 +51,22 @@
       end
     end
 
+    factory :partners_user do
```

Comment:
Why was this renamed?

Reply:
There were two copies, one called :partners_user defined in factories/partners_user.rb and one called :partner_user defined in factories/partner/user.rb

The codebase used both so I just did a coin flip and picked one to rename all of them to